lib/cmd/cmd.go

```diff
@@ -168,3 +167,30 @@ func bindFlags(cmd *cobra.Command, v *viper.Viper) error {
 
 	return lastErr
 }
+
+// wrapRunCmd wraps the "app run" command to custom fatal error log and silence cobra output.
+func wrapRunCmd(cmd *cobra.Command) {
+	runCmd := getRunCmd(cmd)
+	SilenceErrUsage(runCmd)
+	runFunc := runCmd.RunE
```

Comment:
lacking a bit of context but is it safe to assume RunE won't be nil?

Reply:
it would be a misconfiguration be us, but sure will add a check